tests/foreman/cli/test_activationkey.py

```diff
@@ -46,6 +44,8 @@
 from robottelo.constants import PRDS
 from robottelo.constants import REPOS
 from robottelo.constants import REPOSET
+from robottelo.host_helpers.cli_factory import CLIFactory
```

Comment:
No importing, the CLIFactory class object `cli_factory` is accessible from satellite object and hence it should should have been called as :

`<sat_obj>.cli_factory.<helper_method>`